src/biokbase/narrative/upa.py

```diff
@@ -5,20 +5,19 @@
 """
 
 import re
-
-from .app_util import system_variable
+from .system import system_variable
 
 external_tag = "&"
 
 
```

Comment:
The various typing things also weren't exactly intended to pop into this PR, but they're benign and generally helpful.